SpringParking-parkingBranch/src/test/java/pl/springDemo/demo/userPaymentTest/UserPaymentRepoTest.java

```diff
@@ -0,0 +1,57 @@
+package pl.springDemo.demo.userPaymentTest;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.springDemo.demo.user.UserType;
+import pl.springDemo.demo.userPayment.UserPayment;
+import pl.springDemo.demo.userPayment.UserPaymentRepo;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class UserPaymentRepoTest {
```

Comment:
Czemu tylko dwie metody z UserPaymentRepo są przetestowane?

Reply:
Tak jak wszystkie braki w ilości testów, wynika to z faktu, że nie byłam pewna jak duże powinno być pokrycie testowe kodu.